cvat/apps/documentation/user_guide.md

```diff
@@ -182,9 +184,18 @@ Go to the [Django administration panel](http://localhost:8080/admin). There you
 
     **Stop frame**. Frame on which video in task ends.
 
-    **Frame Filter**. Use this option to filter video frames.
+    **Frame Step**. Use this option to filter video frames.
     For example, enter ``step=25`` to leave every twenty fifth frame in the video. Use this option on video files only.
```

Comment:
Could you please check that we still need step=25 instead of 25?

Reply:
@nmanovic  Yes, 25 is too much. Maybe make this text:
"For example, enter `` step = 4`` to leave every fours frame in the video. Use this option to reduce the frame rate in a video files only."

Reply:
I mean that the correct value for the field should  be `25` not `step=25` in accordance with its new name.

Reply:
@bsekachev , could you please clarify?

Reply:
@TOsmanov @nmanovic 
Yes, now correct value is just a number. For example in this case ``25`` instead of ``step=25``